repose-aggregator/src/docs/asciidoc/services/metrics.adoc

```diff
@@ -99,3 +114,63 @@ meter.mark(); // <3>
 <2> Creates a `Meter` with the name `prefix.parent.child.summingMeter`.
 <3> Marks the `Meter` as well as both `ACROSS ALL` `Meter` s of the ancestral `SummingMeterFactory` s.
     This means that all of the following `Metric` s are marked: `prefix.parent.child.summingMeter`, `prefix.parent.child.ACROSS ALL`, and `prefix.parent.ACROSS ALL`.
+
+=== Metrics Directory
+
+The following lists attempt to aggregate all of the metrics being reported to this service by various components.
+
+==== Filters
+
+* <<../filters/api-validator.adoc#_metrics, API Validator Filter>>
+* <<../filters/destination-router.adoc#_metrics, Destination Router Filter>>
+* <<../filters/header-normalization.adoc#_metrics, Header Normalization Filter>>
+* <<../filters/uri-normalization.adoc#_metrics, URI Normalization Filter>>
+* <<../filters/versioning.adoc#_metrics, Versioning Filter>>
+
+==== Miscellaneous
+
+[cols="4", options="header,autowidth"]
+|===
+|Component
+|Metric Type
+|Metric Name
+|Description
+
+| Power Filter
+| Meter
+| `org.openrepose.core.ResponseCode.Repose.<response-code>`
+| Counts the number of responses with a `<response-code>` status code returned to the client.
+`<response-code>` is the HTTP status code in the response before being sent to the client (i.e., after Repose processing has occurred).
+
+| Power Filter Chain
+| Timer
+| `org.openrepose.core.FilterProcessingTime.Delay.<name>`
+| Tracks the amount of time it takes for a component named `<name>` to complete its processing.
+`<name>` is either the name of a filter or `route`.
+`route` indicates that the component being timed is the routing component, which includes forwarding a request to the origin service and waiting on a response.
+
+| Power Filter Router
+| Meter
+| `org.openrepose.core.ResponseCode.<endpoint>.<response-code>`
+| Counts the number of responses with a `<response-code>` status code returned by `<endpoint>`.
+`<response-code>` is the HTTP status code in the response from `<endpoint>`.
+`<endpoint>` is the endpoint that the request was made to.
+
+| Power Filter Router
+| Meter
+| `org.openrepose.core.ResponseCode.ALL ENDPOINTS.<response-code>`
```

Comment:
We really did. However, thanks for calling this out. We actually case those words differently from how we case `ACROSS ALL`. This should say `All Endpoints`.

Because that is clear.